pom.xml

```diff
@@ -234,6 +231,11 @@
                 <![CDATA[
                   {
                     "revapi" : {
+                      "semver": {
+                        "ignore": {
+                          "enabled": true
```

Comment:
As for why it fails the build the first time - I don't think there is a way in maven to do it in go - essentially revapi needs the build to build the artifact before it can compare it with the old version at which point it is too late to reset the version in the pom.